feature/qos/otg_tests/ingress_police_nhg/README.md

```diff
@@ -0,0 +1,363 @@
+# TE-18.2 QoS scheduler with 1 rate 2 color policer, classifying on next-hop group
+
+## Summary
+
+Use the gRIBI applied ip entries from TE-18.1 gRIBI. Configure an ingress scheduler
+to police traffic using a 1 rate, 2 color policer. Configure a classifier to match
+traffic on a next-hop-group.  Apply the configuration to a VLAN on an aggregate
+interface.  Send traffic to validate the policer.
+
+## Topology
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Test setup
+
+Use TE-18.1 test environment setup.
+
+## Procedure
+
+### TE-18.2.1 Generate and push configuration
+
+* Generate config for 2 classifiers which match on next-hop-group.
+* Generate config for 2 forwarding-groups mapped to "dummy" input queues
```

Comment:
Darren,, Please elaborate what does this mean.

Reply:
This is a summary in English for the [OpenConfig ingress policing syntax defined in the readme](https://github.com/openconfig/featureprofiles/pull/3383/files#diff-f219a5b1de6d0179fdad45ecad55c22e7758fc4bd4c729e0e0c37f6104490179R32-R182)

[Additional description is in this comment](https://github.com/openconfig/featureprofiles/pull/3383#issuecomment-2364695288)

I plan to create more extensive documentation on this at openconfig.net in the near future.